include/UnixStamp.hpp

```diff
@@ -18,34 +20,39 @@
 #define APPROXIMATE_DAYS_IN_YEAR 365
 #define FOUR_YEAR_CICLE_YEARS 4
 #define YEARS_IN_CENTURY 100
-#define DAYS_AFTER_FIRST_MARCH 153
+#define DAYS_BETWEEN_MARCH_AND_AUGUST 153
 
 class UnixStamp
 {
 private:
-    time_t unix;
+    // Both timestamps are saved in GMT+0
+    unixstamp unix;
+    civil_time time;
```

Comment:
I have a few difficult questions about this design, but that's outside of the scope of this p/r.

Reply:
Ok, I'm open for a discussion.